doc/source/tutorial/tutorial_IO.ipyml

```diff
@@ -132,7 +145,8 @@ cells:
 - markdown: |
     ##### Basic Usage
     
-    ``open_excel`` must be used with the Python keyword ``with``. If the Excel file doesn't exist, the argument ``overwrite_file`` must set to True. The method ``save`` must be called to actually write data in the Excel file. 
+    ``open_excel`` should be used with the Python keyword ``with``, which ensures that the file is properly closed even if an error occurs.
+    If the Excel file doesn't exist, the argument ``overwrite_file`` must set to True. The method ``save`` must be called to actually write data in the Excel file.
```

Comment:
s/argument overwrite_file/overwrite_file argument/
s/must set/must be set/
s/method save/save method/